### PR TITLE
Fixed cluster context switch for rke :PA-1773

### PR DESCRIPTION
**Pull request by snigdha-px**
**What this PR does / why we need it**:
In method SetClusterContext , if the clusterConfigPath is empty, setting the context to source cluster
**Which issue(s) this PR fixes** (optional)
Closes #https://portworx.atlassian.net/browse/PA-1773

**Special notes for your reviewer**:
https://jenkins.pwx.dev.purestorage.com/job/portworx-backup/job/system-tests/job/basic-system-test/job/px/job/NFS/job/PX-Cloud-RKE-NFS/176/

### CODE REVIEW

**Review by kshithijiyer-px — APPROVED** [review]
LGTM

File: tests/common.go [hunk 1]
```diff
@@ -2902,9 +2902,15 @@ func SetClusterContext(clusterConfigPath string) error {
...
 	if os.Getenv("CLUSTER_PROVIDER") == drivers.ProviderRke {
 		if !strings.HasPrefix(clusterConfigPath, "/tmp/") {
-			return fmt.Errorf("clusterConfigPath cannot be an empty string for %s cluster provider", drivers.ProviderRke)
+			if clusterConfigPath == "" {
+				err = Inst().S.(*rke.Rancher).UpdateRancherClient("source-config")
+				if err != nil {
+					return fmt.Errorf("failed to update rancher client for default source cluster context with error: [%v]", err)
+				}
+			}
+			return fmt.Errorf("invalid clusterConfigPath: %s for %s cluster provider", clusterConfigPath, drivers.ProviderRke)
```
Comment (mkoppal-px) [inline tests/common.go:2911]: Shouldn't this come under an else block?

Comment (snigdha-px) [reply tests/common.go:2913]: done

**Review by mkoppal-px — APPROVED** [review]
LGTM

File: tests/common.go (hunk 1 above)
Comment (snigdha-px) [reply tests/common.go:2912]: done

**Pull request merged**